src/templates/static/affiliates/signup.jsx

```diff
@@ -148,6 +149,13 @@ const Signup = () => (
                             </div>
                             <p className='mobile-text-small'>{it.L('All commissions are credited into your account by the 15th of every month.')}</p>
                         </TabContent>
+                        <TabContent id='cpa_eu_only'>
+                            <h3 className='gr-padding-20'>
+                                <span>{it.L('Cost per acquisition (CPA) for EU affiliates only')}</span>
+                            </h3>
+                            <p className='mobile-text-small'>{it.L('Earn USD 100 in commission for each successful referral. Your referred client must open a <span className=\'bold\'>real money investment account</span> through your unique affiliate link and deposit a total of USD 100 or more (one-time or accumulative) in the account. This commission plan is only available to affiliates based in the EU.')}</p>
```

Comment:
instead of `<span className=\'bold\'>` why not use `<strong>`? also please don't include HTML tags in translation strings, better use placeholders for them